### PR TITLE
Qblox classifier

### PR DESCRIPTION
**Pull request by Edoardo-Pedicillo**
This PR benchmarks the Qblox classifier asked in #305.
Checklist:
- [x] Reviewers confirm new code works as expected.
- [x] Tests are passing.
- [x] Coverage does not decrease.
- [x] Documentation is updated.

### CODE REVIEW

**Comment by aorgazf** [comment]
Hi @Edoardo-Pedicillo,
Could you please elaborate a bit what is the purpose of this PR.

**Comment by Edoardo-Pedicillo** [comment]
In this PR I am adding the Qblox classification method to the other classifiers, in this way we can compare its performances with the others. I am also correcting the Ramiro's method in src/qibocal/fitting/classifier/qubit_fit.py

File: extras/classification/play.py [hunk 1]
```diff
@@ -5,15 +5,16 @@
...
-base_dir = Path("_results3")
+base_dir = Path("results")
 base_dir.mkdir(exist_ok=True)
 
 for qubit in range(1, 5):
     print(f"QUBIT: {qubit}")
     qubit_dir = base_dir / f"qubit{qubit}"
     table, y_test, x_test = run.train_qubit(
-        data_path, base_dir, qubit, classifiers=["qubit_fit"]
+        data_path, base_dir, qubit  # , classifiers=["qblox_fit"]
```
Comment (andrea-pasquale) [inline extras/classification/play.py:15]: Remove commented code

File: extras/classification/play.py [hunk 2]
```diff
@@ -5,15 +5,16 @@
...
     )
     run.dump_benchmarks_table(table, qubit_dir)
+
     plots.plot_table(table, qubit_dir)
-    plots.plot_conf_matr(y_test, qubit_dir)
+    # plots.plot_conf_matr(y_test, qubit_dir)
```
Comment (andrea-pasquale) [inline extras/classification/play.py:20]: Remove commented code

File: src/qibocal/fitting/classifier/qblox_fit.py [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+    """
+    return {}
+
+
+normalize = identity
+
+
+@dataclass
+class QbloxFit:
+    r"""This class deploys the Qblox qubit state classifier.
```
Comment (andrea-pasquale) [inline src/qibocal/fitting/classifier/qblox_fit.py:37]: Here I would add a link to the qblox documentation.

**Review by andrea-pasquale — COMMENTED** [review]
Thanks @Edoardo-Pedicillo.
Find below a few minor comments.

**Comment by andrea-pasquale** [comment]
Thanks @Edoardo-Pedicillo.
Can we do something about the coverage?

**Comment by Edoardo-Pedicillo** [comment]
Now the coverage increases. @aorgazf, when you have time, could you review the code? Thanks.

**Review by andrea-pasquale — APPROVED** [review]

**Pull request merged**